CMakeLists.txt

```diff
@@ -202,6 +202,13 @@ if (USE_DTRACE)
   add_definitions("-DUSE_DTRACE=1")
 endif ()
 
+# enable code coverage
+option(USE_COVERAGE "enable gcov code coverage" OFF)
+if (USE_COVERAGE)
+  add_compile_options(-fno-stack-protector -fprofile-arcs -ftest-coverage)
```

Comment:
Why `-fno-stack-protector` for coverage?

Reply:
I did not question
https://github.com/arangodb/oskar/blob/master/scripts/buildAlpine6.fish#L58

if its not needed, we can remove it.

Reply:
This is a copy of current options unrelated to functional changes of this PR.